docs_user/modules/openstack-planning.adoc

```diff
@@ -204,6 +204,23 @@ are eligible to run the different OpenStack services.  Refer to the xref:node-se
 selector] to learn more about using labels to define
 placement of the OpenStack services.
 
+== TLS Everywhere
+
+Note: The assumption is, that the new deployment will adopt the settings from the
+old deployment, so in case TLS Everywhere is disabled, it won't be enabled on
+the new deployment.
+
+If the Director deployment was deployed with TLS Everywhere, FreeIPA (IdM) is used
+to issue certificates for the OpenStack services. Certmonger, a client process which
+is installed on all hosts, interacts with FreeIPA (IdM) to request, install, track
+and renew these certificates.
+
+The new Operator based deployment uses the cert-manager operator to issue, track
+and renew the certificates.
+
+Because the same root CA is used to generate new certificates, the currently used chain
```

Comment:
Replace this last sentence with:

The TLS everywhere document (link) describes how to extract the root CA from IdM in the source environment and import it for use by certmanager in the new environment.  Because the same root CA is used to issue new certificates, we can minimize disruption on the compute nodes because the currently used chain of trust does not need to be modified.